source/index.html

```diff
@@ -1,12 +1,93 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+  <link rel="stylesheet" href="styles/styles.css">
+</head>
+<body class="page__body">
+  <header class="header">
+    <nav class="header__nav nav">
+      <a class="nav__logo" href="#">
+        <img src="img/images/logo-1.svg" width="70" height="59" alt="логотип Cat Energy">
+        <img src="img/images/logo-2.svg" width="101" height="18" alt="cat energy">
+      </a>
+      <button class="header__button-header" type="button"><span class="visually-hidden">Открыть меню</span></button>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="main-container__wrapper wrapper">
+      <h1 class="wrapper__title">Функциональное питание для котов</h1>
+      <p class="wrapper__text">Занялся собой? Займись котом!</p>
+      <img src="img/images/general-img.jpg" width="280" height="270" alt="Корм для котов">
+      <button class="wrapper__button" type="button"><span>Подобрать программу</span></button>
+    </section>
 
-  </body>
+    <div class="main-container__advantages advantages">
+      <article class="advantages__slimming">
+        <img src="img/images/cat-index-1.svg" alt="кот">
+        <h2 class="advatages__title">Похудение</h2>
+        <p class="advantages__text">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim
+          поможет вашему питомцу сбросить лишний вес.</p>
+      </article>
+      <article class="advantages__weight-gain">
```

Comment:
Что конкретно описывается в элемента не важно, выглядят они одинаково.

```suggestion
      <article class="advantages__item">
        <img src="img/images/cat-index-1.svg" alt="кот">
        <h2 class="advatages__title">Похудение</h2>
        <p class="advantages__text">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim
          поможет вашему питомцу сбросить лишний вес.</p>
      </article>
      <article class="advantages__item">
```